client/scss/components/_listing.scss

```diff
@@ -299,6 +307,7 @@ ul.listing {
     color: theme('colors.text-button-outline-default');
     display: block;
     text-align: center;
+    line-height: 3rem;
```

Comment:
I guess this could go either way really... I took the plunge of setting top alignment globally (in d758bd41d9be8b1dc8dda355673cb0f72bf99e7e) so that tables with one overly tall column (e.g. the People snippet index on bakerydemo, which has a thumbnail image in it) don't end up shifting the other cells' contents weirdly - and also because we were setting it on an ad-hoc basis on a load of tables, to the point where it seemed we were fighting against a bad default. I guess this was me following that design decision through to its conclusion - but maybe this is a sensible exception to the rule, since it's treating the table cell as more of a button than a content element. (On the other hand, the new design - not implemented yet as it requires being able to calculate the descendant count - does bring it back into line with the 'link' style elsewhere in the table. Hmm...)

Reply:
👍 what we have right now seems reasonable to me for this iteration of the designs. We should probably ask Ben to design a wrapping version and see where he takes it.